definitions/npm/express_v4.x.x/flow_>=v0.25.x/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: mixed & {[name: string]: string};
```

Comment:
Mmm.. yep, I'm thinking that it is as you say, I will change it tomorrow, or in the worst case, on Thursday.